pkgs/build-support/fetchcipd/builder.sh

```diff
@@ -0,0 +1,5 @@
+source $stdenv/setup
+
+cipd ensure -root $NIX_BUILD_TOP -ensure-file $ensureFile
+mkdir -p $out
+cp -r $(readlink -e $NIX_BUILD_TOP/src)/* $out
```

Comment:
or simply

```sh
cipd ensure -root $out -ensure-file $ensureFile
```

problem with `cp dir1/* dir2/`: dotfiles are not copied (fix: shopt -s dotglob)

Reply:
Done, I had issues earlier during my own testing where the `$out` wouldn't exist when being used in the `$src` stage for the `flutter-engine` and this had apparently fixed it. Also `cipd`, doesn't do top-level dotfiles because it creates a directory for each package.

Reply:
Doing `cipd ensure -root $out -ensure-file $ensureFile` creates 2 files and 1 directory. `README`, `update.sh`, and `src`. Turns out we do have to use `$NIX_BUILD_TOP`.